receiver/opencensusreceiver/testdata/config.yaml

```diff
@@ -47,6 +47,9 @@ receivers:
     # This receiver is disabled and won't receive any data.
     disabled: true
 
+  opencensus/cors:
```

Comment:
Can you add a few lines about what this does in this file? Ie: that it will accept requests from a.test.com but not wwwtest.com for ex?